test/views/html/PensionDetailsSconSpec.scala

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
Is it flagged somewhere for us to delete this file prior to final merge/release?

Reply:
on list for clean up